bootstrap/bootstrap.ps1

```diff
@@ -52,9 +56,12 @@ if ($LastExitCode -ne 0)
 
 # create a junction to the shared FX version directory. this is
 # so we have a stable path to dotnet.exe regardless of version.
-$junctionTarget = Join-Path $toolsLocalPath "dotnetcli\shared\Microsoft.NETCore.App\1.0.0"
```

Comment:
Updated, I've change it so the new behavior is that dotnet-install will optionally write out a Microsoft.NETCore.App.version file to a supplied path (in build_tools in CLI so it's already in .gitignore).  bootstrap will then grab the version out of that file and use that version for the symlink.